README.md

```diff
@@ -20,8 +20,7 @@ We assume that you use the default `pi` user.
 
 Just write...
 
-```
-sudo apt install git
```

Comment:
Git isn't installed by default in Raspbian Lite, I think it is better to keep it.

Reply:
Well, ok, didn't know.